tests/test_rope.py

```diff
@@ -57,12 +57,11 @@ def test_rope(lit_llama, orig_llama) -> None:
 
     freqs_cis = orig_llama.precompute_freqs_cis(n_embed // n_head, seq_len)
     llama_rope_cache = lit_llama.build_rope_cache(seq_len, n_embed // n_head, dtype=x.dtype, device=x.device)
-    assert torch.equal(freqs_cis, llama_rope_cache)
+    torch.testing.assert_close(freqs_cis, torch.view_as_complex(llama_rope_cache))
 
     llama_x_rope = lit_llama.apply_rope(x.transpose(1, 2), llama_rope_cache).transpose(1, 2)
     orig_llama_x_rope, _ = orig_llama.apply_rotary_emb(x, x, freqs_cis)
-
-    assert torch.equal(llama_x_rope, orig_llama_x_rope)
+    torch.testing.assert_close(llama_x_rope, orig_llama_x_rope)
 
     # For posterity, we show here that our older implementation we initially wanted to use
     # is not numerically equivalent to Meta's rope implementation
```

Comment:
The old implementation in this file, this comment and the subsequent check can now be removed

Reply:
OK!